src/Level.java

```diff
@@ -1,9 +1,50 @@
-public class Level extends Game{
+public class Level {
 
     String label;
-    public Level (String label){
+    int height = 7; //Enter number between 6 and 20
+    char[][] map = createEmptyMap(height);
+    Game game;
+
+    //this could be also done by "createGame" method
+    Walls walls = new Walls(map, 0, '+');
+    Hero hero = new Hero(map, 'H');
+    Stone stone = new Stone(map, 'O');
+    Hole hole = new Hole(map, 'X');
+    Trap trap = new Trap(map, '8');
+
+
+    public Level(String label) {
```

Comment:
A, czyli u Ciebie Level tworzy Game, ciekawe.
Jak dla mnie to Game brzmi jak 1sza wartswa, coś co ma Game Loop i konsolę, level, map i te elementy byłyby w hierarchii niżej (nawet jak tworzone przed Game w main żeby je gdzieś powstrzykiwać)